ChatCore/Utils/TaskManager.swift

```diff
@@ -11,6 +11,44 @@ import BackgroundTasks
 
 // MARK: Helper class to automatically manage closures by applying various attributes
 final class TaskManager {
+    class TaskCache {
+        typealias Closure = IdentifiableClosure<TaskCompletionResultHandler, Void>
+        
+        private var storage: [Closure: Set<TaskAttribute>] = [:]
```

Comment:
what about lazy var?

Reply:
Is it necessary if it's just a dictionary?

Reply:
I added `lazy` keyword, but still - it's an empty dictionary, so it should be fine even without it, right?